### DISCUSSION TITLE
FlowULTRA / Flow 120/C120

### DISCUSSION THREAD

**Discussion by lollox80** [DanRobo76/VMC-HELTY-FLOW#13] [Generale]
Ciao, bellissimo progetto. Sto ristrutturando casa ed ho un open space per la zona giorno di 55 m2  (143 m3) stavo pensando ad una VMC helty. Visto la grandezza del ambiente sono indirizzato su FlowULTRA / Flow 120/C120, sai se sono integrabili con il tuo pacchetto?

**Comment by DanRobo76** [comment]
Non ho idea, ma a sensazione direi di sì .. prova a sentire helty ciao



Il mar 5 nov 2024, 10:07 lollox80 ***@***.***> ha scritto:
